CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Zulip-terminal Changelog
 
+## Next Release
+
+Expected to be a point release, this contains much of the remaining work from last year's GSoC students, and various other contributors - thanks to you all! (180+ commits)
+
+If you've been waiting for autocomplete, popup notifications, mentions, better themes & search...this release is for you!
+
+### HIGHLIGHTS
+- Autocomplete users/groups/streams/emoji when composing messages, given a prefix (<kbd>ctrl</kbd>+<kbd>f</kbd> / <kbd>ctrl</kbd>+<kbd>r</kbd>)
+- Popup notifications for private messages & mentions (NOTE: disabled by default; tools may be required on WSL/Linux)
+- See details of messages in which you are mentioned - narrow to them using <kbd>#</kbd>
+- All themes now considered usable (light/blue much improved)
+- Search for messages only within a specific narrow, not just within all messages
+- Support added for Python 3.8 and PyPy
+- Sample Dockerfiles added, to enable building containers to run from any docker-enabled system
+
+### Interactivity improvements
+- Improved topic list handling (search topics, topics update)
+- Support /me slash command
+- Toggle display of stream description using <kbd>i</kbd>
+- Enable/disable autohide setting at startup
+- Extend support for vim-like navigation (scrolling)
+
+### Visual improvements
+- Add support to render tables in messages
+- Improve rendering of links in messages
+- Categorize and further reorganize help menu
+- Show the year of a message, if it is not the current year
+- Show server name & description in the title bar
+
+### Important bugfixes
+- On changing to a narrow which contains the focused message, retain focus on that message
+- Message search results reset between searches
+- Improved unread count handling
```

Comment:
One more bug which existed in the previous release was (https://github.com/zulip/zulip-terminal/pull/462); I happened to install v0.4.0 for testing and ran into this within 5 minutes. I think folks would be glad to hear that the UI is no longer distorted during some startups.

Reply:
Right, good catch! I've marked this as #456, as that was the bug itself?